packages/search-spa/set-env.ts

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+const writeFile = fs.writeFile;
+
+const targetPath = './src/environments/environment.ts';
+
+require('dotenv').config();
+
+const envConfigFile = `export const environment = {
+   API_URL: '${process.env.API_URL}',
```

Comment:
String template interpolation is not needed if you can use the variable as it is. All the variables in `process.env` are already strings by default.

Also, if the env variables is not defined, it would set it to `"undefined"` as a string. Which could cause problems. So just use the variables as it is.

```suggestion
   API_URL: process.env.API_URL,
```